audio/out/ao.c

```diff
@@ -54,10 +54,14 @@ extern const struct ao_driver audio_out_wasapi;
 extern const struct ao_driver audio_out_pcm;
 extern const struct ao_driver audio_out_lavc;
 extern const struct ao_driver audio_out_sdl;
+extern const struct ao_driver audio_out_aaudio;
 
 static const struct ao_driver * const audio_out_drivers[] = {
 // native:
-#if HAVE_ANDROID
+#if HAVE_AAUDIO
+    &audio_out_aaudio,
```

Comment:
Should this has higher priority than audiotrack?

Reply:
Probably not since it's not supported on older devices

Reply:
That probably doesn't matter, it'll just fallback to audiotrack once aaudio fails. The question is on devices that support both, should this be preferred?

Reply:
It depends. AAudio is definitely more performance, but because AAudio didn't gain the ability to passthrough audio to HDMI or S/PDIF until Android 14 which is too new. So if users with a Android TV system (many of them running versions much older than Android 14) needs that then it won't automatically fallback since the audio will just be transcoded.

Reply:
Maybe in a few years this can be the default.